models/stg_recharge__charge_line_item.sql

```diff
@@ -25,18 +25,18 @@ final as (
         subscription_id,
         vendor,
         title,
-        property_title as title_v2,
+        property_title as title_v2, 
```

Comment:
Why is this renamed to title_v2?

Reply:
`property_title` is the recharge title, and `title` is the Shopify title, so was thinking that would differentiate, but upon 2nd thought, it's really not necessary, so I reverted back to the original title to avoid confusion.